test/e2e/cypress/integration/about.js

```diff
@@ -1,42 +1,50 @@
-describe('User account page', () => {
+describe("User account page", () => {
   before(() => {
-    cy.navigateToItem('About');
-    cy.url().should('include', '/about');
+    cy.navigateToItem("About");
+    cy.url().should("include", "/about");
   });
 
-  it('should have the correct page title', () => {
-    cy.get('h2.text-5xl').should('contain', 'About Trento Console');
+  it("should have the correct page title", () => {
+    cy.get("h2.text-5xl").should("contain", "About Trento Console");
   });
 
-  it('should show the correct flavor', () => {
-    cy.get('div')
-      .contains('Trento flavor')
-      .next()
-      .should('contain', 'Community');
+  it("should show the correct flavor", () => {
+    let flavour = "Community";
+    if (Cypress.env("flavour")) {
+      flavour = Cypress.env("flavour");
+    }
+    cy.get("div").contains("Trento flavor").next().should("contain", flavour);
   });
 
-  it('should show the correct server version', () => {
-    cy.exec(`cd ${Cypress.env('project_root')} && mix version`).then(
-      ({ stdout: version }) => {
-        cy.get('div')
-          .contains('Server version')
-          .next()
-          .should('contain', version);
-      }
-    );
+  it("should show the correct server version", () => {
+    let version;
+    if (Cypress.env("version")) {
+      version = Cypress.env("version");
+    } else {
+      cy.exec(`cd ${Cypress.env("project_root")} && mix version`).then(
+        ({ stdout: out_version }) => {
+          version = out_version;
+        }
+      );
+    }
```

Comment:
This expectation will run against an empty `version` variable due to the asyncronous nature of JS and Node on which Cypress is based. What you could do is turning this function into an `async` one and `await` for the `cy.exec` output. You can ping me anytime.

Reply:
What about just catching the missing `mix` binary error on the `cy.exec` and attempting to get the environment variable in that case? you'd have to run the `cy.get` in the if/else which is a bit of a smell... but I'm not a huge fan of how we are using `mix version` here anyways so I wouldn't invest too much time in it

Reply:
> catching the missing mix binary error 
Is it not more safe/clear to let the user decide, using the env variable, what to do? Is it not interesting (let the test to fails) to know if `mix version` is not able to properly run?